src/runtime_src/driver/xclng/tools/xbutil/xbutil.cpp

```diff
@@ -1158,131 +1164,129 @@ int xcldev::xclValidate(int argc, char *argv[])
     return 0;
 }
 
-static int getEccMemTags(const pcidev::pci_device *dev,
-    std::vector<std::string>& tags)
-{
-    std::string errmsg;
-    std::vector<char> buf;
-
-    dev->user->sysfs_get("", "mem_topology", errmsg, buf);
-    if (!errmsg.empty()) {
-        std::cout << errmsg << std::endl;
-        return -EINVAL;
-    }
-    const mem_topology *map = (mem_topology *)buf.data();
-
-    if(buf.empty() || map->m_count == 0) {
-        std::cout << "WARNING: 'mem_topology' not found, "
-            << "unable to query ECC info. Has the xclbin been loaded? "
-            << "See 'xbutil program'." << std::endl;
-        return -EINVAL;
-    }
-
-    // Only support DDR4 mem controller for ECC status
-    for(int32_t i = 0; i < map->m_count; i++) {
-        if(map->m_mem_data[i].m_type != MEM_DDR4 || !map->m_mem_data[i].m_used)
-            continue;
-        tags.emplace_back(
-            reinterpret_cast<const char *>(map->m_mem_data[i].m_tag));
-    }
-
-    if (tags.empty()) {
-        std::cout << "No supported ECC controller detected!" << std::endl;
-        return -ENOENT;
-    }
-
-    // See if xclbin contains ECC base addresses for supported in-use DDR type
-    unsigned onoff = 0;
-    dev->mgmt->sysfs_get(tags[0], "ecc_enabled", errmsg, onoff);
-    if (!errmsg.empty()) {
-        std::cout << "No supported ECC controller detected!" << std::endl;
-        return -ENOENT;
-    }
-
-    return 0;
-}
-
-static int eccStatus2String(unsigned int status, std::string& str)
+static int getEccMemTags(const pcidev::pci_device *dev,	
```

Comment:
Changes related to the ECC code looks strange. Do we know why they show up?

Reply:
I am not sure why they show up. I checked each and every line and I couldn't see any difference. They are exactly the same